anchor-reference-server/src/main/java/org/stellar/anchor/reference/config/IntegrationAuthSettings.java

```diff
@@ -1,13 +1,11 @@
 package org.stellar.anchor.reference.config;
 
 import lombok.Data;
-import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 import org.stellar.anchor.config.IntegrationAuthConfig.AuthType;
 
 @Data
 @Configuration
-@ConfigurationProperties(prefix = "integration-auth")
```

Comment:
Why are you removing this? Won't it be impossible for the reference server to pick up the configuration?

Reply:
That's right. I have added this back.